examples/with_openai/setup.py

```diff
@@ -5,8 +5,9 @@
     packages=find_packages(exclude=["with_openai_tests"]),
     install_requires=[
         "dagster",
+        "dagster-aws",
         "dagster-openai",
-        "dagster-cloud",
+        # "dagster-cloud", # Uncomment when using Dagster+
```

Comment:
i think we can leave it here. if ppl are not using dagster+, it will just be installed but unused right?

Reply:
Yes, that's not a problem. I commented it because Pyright was yelling because of a version conflict - we can't rebuild the pins otherwise.